mod-inventory/src/main/resources/domain/mod-inventory/features/inventoryFeatureTest.feature

```diff
@@ -5,13 +5,32 @@ Feature: inventory sample
     * callonce login testUser
     * configure headers = { 'x-okapi-tenant':'#(testTenant)','Content-Type': 'application/json', 'x-okapi-token': '#(okapitoken)', 'Accept': 'application/json, text/plain' }
 
-
-    Scenario: new Instance creation
+  Scenario: new Instance, Holding, Item creation
     * def createInstanceRequest = read('samples/createInstance.json')
       Given path 'inventory/instances'
       And request createInstanceRequest
       When method POST
       Then status 201
+      * def location = responseHeaders['Location'][0]
+      * def referenceId = location.substring(location.lastIndexOf('/') + 1)
```

Comment:
Given that this test involves multiple references. I think it could be less ambiguous to name this `instanceId` rather than `referenceId`

```suggestion
      * def referenceId = location.substring(location.lastIndexOf('/') + 1)
```